pkg/kubectl/cmd/cmd.go

```diff
@@ -30,30 +30,35 @@ import (
 
 const (
 	bash_completion_func = `# call kubectl get $1,
-__kubectl_namespace_flag()
+__kubectl_override_flag_list=(kubeconfig cluster user context namespace server)
+__kubectl_override_flags()
```

Comment:
is that really something generic that you set the context (in the non-k8s sense) of a command? I don't really see that in cobra either.